<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoManager</class>
 <widget class="QWidget" name="VideoManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Video</string>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <widget class="QListWidget" name="videoListWidget">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>20</y>
     <width>321</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addVideoStepBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>AddVideoStep</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="ImageRdBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>AsImage</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="FullScreenRdBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>FullScreen</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>340</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="addVideoBtn">
      <property name="text">
       <string>AddVideo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="removeVideoBtn">
      <property name="text">
       <string>RemoveVideo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
